x-pack/legacy/plugins/watcher/__jest__/client_integration/watch_create_threshold.test.tsx

```diff
@@ -192,39 +165,30 @@ describe.skip('<ThresholdWatchEdit /> create route', () => {
           expect(exists('watchActionsPanel')).toBe(true);
         });
 
-        describe('watch conditions', () => {
-          beforeEach(async () => {
-            const { form, find, component } = testBed;
+        // Looks like there is an issue with using 'mockComboBox'.
+        describe.skip('watch conditions', () => {
```

Comment:
Can you update https://github.com/elastic/kibana/issues/52620 and mention this is the only remaining skipped test.